apps/rush-lib/src/utilities/Stopwatch.ts

```diff
@@ -98,4 +93,22 @@ export class Stopwatch {
       return `${totalSeconds.toFixed(2)} seconds`;
     }
   }
+
+  /**
+   * Get the duration in seconds.
+   */
+  public get duration(): number {
+    return this._getTotalSeconds();
+  }
+
+  private _getTotalSeconds(): number {
```

Comment:
is it because the call to _getTime?

---
In reply to: [117848818](https://github.com/Microsoft/web-build-tools/pull/207#discussion_r117848818) [](ancestors = 117848818)